playwright/e2e/amplitude-autocapture.spec.ts

```
@@ -0,0 +1,335 @@
+    }
+
+    // Navigate to a page to trigger autocapture initialization
+    // and wait for the Amplitude request to be sent
+    await Promise.all([
+      // Wait for Amplitude API request
+      page.waitForRequest(
+        (request) => {
+          return (
+            (request.url().includes('api.amplitude.com') || request.url().includes('api2.amplitude.com')) &&
+            request.method() === 'POST'
+          );
```

ESLint

[error] Replace `⏎············(request.url().includes('api.amplitude.com')·||·request.url().includes('api2.amplitude.com'))·&&⏎············request.method()·===·'POST'⏎··········)` with `request.url().includes('api.amplitude.com')·||·request.url().includes('api2.amplitude.com'))·&&·request.method()·===·'POST'`

(prettier/prettier)